editor/components/rich-text/index.js

```diff
@@ -852,9 +852,10 @@ export class RichText extends Component {
 			formatters,
 			autocompleters,
 			format,
+			ariaMultiline = true,
```

Comment:
Let's document this prop before it is merged.

Reply:
Documentation was added. Besides documenting I did another update to simplify the code. Now we use the same name as the default aria prop "aria-multiline" but we make it default to true so most RichText users don't need to worry about setting the prop.